db/migrate/20200505224554_add_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTokenToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :token, :string
```

Comment:
I wonder if there are any encryption considerations for this as we discussed & I think it is worth mentioning for guidance at our check-in...

Reply:
Definitely!